resources/kprove.mak

```diff
@@ -78,6 +78,7 @@ $(K_REPO_DIR):
 	git clone $(K_REPO_URL) $(K_REPO_DIR)
 	cd $(K_REPO_DIR) \
 		&& git reset --hard $(K_VERSION) \
+		&& git submodule update --init --recursive \
```

Comment:
@denis-bogdanas why is this needed? we will not need llvm nor haskell backend, right?

Reply:
^^^ @ehildenb